canvas_modules/common-canvas/src/common-properties/controls/abstract-table.jsx

```diff
@@ -320,11 +320,10 @@ export default class AbstractTable extends React.Component {
 			} else {
 				cellContent = (<div className="properties-table-cell-control">
 					<ReadonlyControl
-						control={this.props.control}
+						control={this.props.controller.getControl(propertyId)}
```

Comment:
This was passing in the parent control when we should have been passing in the actual control to be render.  We no longer need columnDef because of that.